skills/sonos-casting/SKILL.md

```
@@ -0,0 +1,391 @@
+├── ☐ Kitchen                     (checkbox - selectable room)
+├── ─────────────────
+├── 🟢 Start Casting              (when NOT casting)
+│   OR
+├── 🔴 Stop Casting               (when casting)
+└── Refresh
+```
+
+### Checkbox Behavior
+
+**When NOT casting:**
+| State | Meaning |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)